frontend/ui/src/pages/Inbox/Messenger/ConversationMetadata/ContactDetails/ContactInfoPoint.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import {ReactComponent as Email} from 'assets/images/icons/email.svg';
```

Comment:
nitpick: as we name those icons with an additional Icon (EmailIcon) so it's more clear that this component "just" a svg.